client/keys/update.go

```diff
@@ -32,10 +33,13 @@ func runUpdateCmd(cmd *cobra.Command, args []string) error {
 	name := args[0]
 
 	buf := bufio.NewReader(cmd.InOrStdin())
-	kb, err := NewKeyBaseFromDir(viper.GetString(flags.FlagHome))
+	var kb keyring.LegacyKeybase
```

Comment:
Why are we still using the legacy keybase for current APIs?

Reply:
The command `update` works only with the legacy keybase. `Update` is no-op with the new keyring

Reply:
I'm wondering whether update could be removed altogether. I think we should, wdyt?

Reply:
We probably should, right?

Reply:
Yes, will do before merging

Reply:
Done, resolving this convo. Thanks!